src/javascript/EditPanel/EditPanelLanguageSwitcher/EditPanelLanguageSwitcher.jsx

```diff
@@ -3,17 +3,20 @@ import {LanguageSwitcher} from '@jahia/design-system-kit';
 import * as PropTypes from 'prop-types';
 import {compose} from '~/utils';
 import {connect} from 'formik';
-import {connect as connectReactRedux} from 'react-redux';
-import {setLanguage} from '~/JContent.redux-actions';
 import {EditPanelDialogConfirmation} from '../EditPanelDialogConfirmation';
+import {useContentEditorConfigContext} from '~/ContentEditor.context';
+import envSwitchLanguages from './EditPanelLanguageSwitcher.env';
 
-// Todo BACKLOG-12437: handle language switcher outside jcontent and gwt
-const EditPanelLanguageSwitcher = ({lang, siteInfo, onSelectLanguage, formik}) => {
+const EditPanelLanguageSwitcher = ({siteInfo, formik}) => {
+    const contentEditorConfigContext = useContentEditorConfigContext();
+    const {lang} = contentEditorConfigContext;
```

Comment:
contentEditorConfigContext contains "lang"?

Reply:
yes, it's part of the config of CE